app/helpers/theme_helper.rb

```diff
@@ -34,6 +34,25 @@ def find(name, prefixes = [], partial = false, keys = [], options = {})
   end
 end
 
+# When rendering collections (i.e. - @foos) with partials, Rails uses this method to build a string
+# such as `foos/foo`. This ultimately references the partial `app/views/foos/_foo.html.erb` for rendering.
+# However, Super Scaffolding class names are namespaced in such a way that `collection` below
+# returns a string with the full path: `scaffolding/absolutely_abstract/creative_concepts/creative_concept`.
+# We only need `creative_concepts/creative_concept`, so we split the string and remove any unncessary elements.
+module ActiveModel
+  module Conversion
+    module ClassMethods
+      def _to_partial_path # :nodoc:
+        @_to_partial_path ||= begin
+          element = ActiveSupport::Inflector.underscore(ActiveSupport::Inflector.demodulize(name))
+          collection = ActiveSupport::Inflector.tableize(name)
+          "#{collection.split("/").last}/#{element}"
+        end
+      end
+    end
+  end
+end
```

Comment:
It seems risky to me to bury a monkey patch in here that redefines partial path generation across a whole Rails app.

Rails has a public API to override which is `to_partial_path` as an instance method, so I'm confused why we can't get away with the code below here?

```ruby
class Scaffolding::AbsolutelyAbstract::CreativeConcept
  def to_partial_path
    "creative_concepts/creative_concept"
  end
end
```

Are there other scaffolding models that need a similar method override?

Reply:
@kaspth You’re 100% right. I ended up running into a bunch of issues with this and have already removed it.

Reply:
@andrewculver ah ok, got it.

@gazayas if you're interested, let me know if you want some pointers on how I think about monkey patches and integrating with Rails — hit me up in the BulletTrain internal Discord chat?

Reply:
Sorry guys! @kaspth, I'll definitely send you a message soon!